release-notes/9.0/os-packages.json

```diff
@@ -416,6 +416,33 @@
                     ]
                 }
             ]
+        },
+        {
+            "name": "FreeBSD",
+            "install-commands":
+            [
+                {
+                    "run-under-sudo": true,
+                    "command-root": "pkg",
+                    "command-parts": ["install", "-A", "{packageName}"]
+                }
+            ],
+            "releases": [
+                {
+                    "name": "FreeBSD 14.1",
+                    "release": "14.1",
+                    "packages": [
```

Comment:
Does this mean that the other packages (like OpenSSL and ca-certs) are installed by default? If not, what's the story on that?